### PR TITLE
changes to hide provider dropdown

### PR DESCRIPTION
**Pull request by M2skills**
## PR Checklist

* [x] Explain your changes, so people looking at the PR know *what* and *why*, the code changes are the *how*.

Moved Provider Instance dropdown outside the current dropdown group.
Made the Provider instance dropdown conditionally visible, hide the dropdown when provider type is HA Cluster.
Updated Headers above the dropdowns.

Alerts Page for HA Alerts 
ha cluster select

Alerts Page For other Provider Types
For other Alerts

### If adding or updating templates:
* [x] post a screenshot of templates and/or gallery changes
* [x] ensure your template has a corresponding gallery entry in the gallery folder
* [ ] If you are adding a new template, add your team and template/gallery file(s) to the CODEOWNERS file. CODEOWNERS entries should be teams, not individuals
* [x] ensure all steps have meaningful names
* [x] ensure all parameters and grid columns have display names set so they can be localized
* [x] ensure that parameters id values are unique __or they will fail PR validation__ (parameter ids are used for localization)
* [x] ensure that steps names are unique __or they will fail PR validation__ (step names are used for localization)
* [x] grep `/subscription/` and ensure that your parameters don't have any hardcoded resourceIds __or they will fail PR validation__
* [x] remove `fallbackResourceIds` and `fromTemplateId` fields from your template workbook __or they will fail PR validation__

### CODE REVIEW

File: Workbooks/SapMonitor2.0/Alerts/Create Alert rule (step 1 of 2)/Create Alert rule (step 1 of 2).workbook [hunk 1]
```diff
@@ -402,10 +410,54 @@
             "timeContext": {
               "durationMs": 86400000
             },
-            "value": "SAP HANA high memory usage percent"
+            "value": "Cluster Primary node switched"
```
Comment (adityaku-ms) [inline Workbooks/SapMonitor2.0/Alerts/Create Alert rule (step 1 of 2)/Create Alert rule (step 1 of 2).workbook:413]: please review this text

File: Workbooks/SapMonitor2.0/Alerts/Create Alert rule (step 1 of 2)/Create Alert rule (step 1 of 2).workbook [hunk 2]
```diff
@@ -402,10 +410,54 @@
...
+            "jsonData": "[\"0\", \"1\", \"2\", \"3\", \"4\"]",
+            "timeContext": {
+              "durationMs": 86400000
+            }
+          }
+        ],
+        "style": "above",
+        "queryType": 12
+      },
+      "name": "parameters - 6"
```
Comment (adityaku-ms) [inline Workbooks/SapMonitor2.0/Alerts/Create Alert rule (step 1 of 2)/Create Alert rule (step 1 of 2).workbook:437]: please update the parameter name

File: Workbooks/SapMonitor2.0/Alerts/Create Alert rule (step 1 of 2)/Create Alert rule (step 1 of 2).workbook [hunk 3]
```diff
@@ -379,13 +379,21 @@
     {
       "type": 1,
       "content": {
-        "json": "### Choose a name, provider instance and severity for the alert"
+        "json": "### Choose a Name, severity for the alert"
```
Comment (adityaku-ms) [inline Workbooks/SapMonitor2.0/Alerts/Create Alert rule (step 1 of 2)/Create Alert rule (step 1 of 2).workbook:382]: "Choose a name and severity for the alert" would be better

File: Workbooks/SapMonitor2.0/Alerts/Create Alert rule (step 1 of 2)/Create Alert rule (step 1 of 2).workbook (hunk 2 above)
Comment (M2skills) [reply Workbooks/SapMonitor2.0/Alerts/Create Alert rule (step 1 of 2)/Create Alert rule (step 1 of 2).workbook]: fixed.

File: Workbooks/SapMonitor2.0/Alerts/Create Alert rule (step 1 of 2)/Create Alert rule (step 1 of 2).workbook (hunk 3 above)
Comment (M2skills) [reply Workbooks/SapMonitor2.0/Alerts/Create Alert rule (step 1 of 2)/Create Alert rule (step 1 of 2).workbook]: fixed

**Review by adityaku-ms — APPROVED** [review]

**Pull request merged**